php/common.php

```diff
@@ -174,7 +176,8 @@ function get_settings($filename = "")
     // отчёты
     $config['reports'] = [
         'auto_clear_messages' => $ini->read('reports', 'auto_clear_messages', 0),
```

Comment:
If I am not mistaken here, those who already have their excluded forum IDs under `reports -> exclude`, will lose the setting.
How about making the following instead?

```suggestion

    // отчёты
    $config['reports'] = [
        'auto_clear_messages' => $ini->read('reports', 'auto_clear_messages', 0),
```